### DISCUSSION TITLE
approved and benefits will be available soon.

### DISCUSSION THREAD

**Discussion by yslaa** [community/community#135677] [GitHub Education]
### Hi everyone,

My application was approved - August 3, 2024 but I still haven't received my benefits yet. It is also said there that the benefits should be received  within 5 days of your approval status and if not raise a discussion on the [GitHub Education community]

**Comment by davevad93** [comment]
Hi @yslaa ,

I noticed you have the `PRO` badge, if the application status label is purple and state "`Congrats! Your request for @yslaa was approved and your benefits are now available.`" that means you have received your benefits.
